### PR TITLE
 commit 14.02.2019 homework

### PR DESCRIPTION
**Pull request by MateAcademy**

### CODE REVIEW

File: src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java [hunk 1]
```diff
@@ -11,31 +11,92 @@
...
+
+    @Override
+    public Set<Order> getAllOrdersJoin() throws SQLException {
+        Set<Order> orders = new HashSet<>();
+        Connection conn = ConnectToDB.getConnection();
+        PreparedStatement stmt = conn.prepareStatement(
+                "SELECT orders.order_num, orders.order_date, orders.mfr, products.product_id, products.mfr_id, "
+                        + "products.description, products.price, products.qty_on_hand,  qty, amount FROM orders "
+                        + "INNER   join products on orders.product = products.product_id");
+        ResultSet rs = stmt.executeQuery();
```
Comment (DadizhaO) [inline src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java:53]: SELECT * looks better

**Review by DadizhaO — APPROVED** [review]
good

File: src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java (hunk 1 above)
Comment (MateAcademy) [reply src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java:53]: ok

File: src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java [hunk 2]
```diff
@@ -11,31 +11,92 @@
...
-
         rs.close();
         stmt.close();
-        connection.close();
+        conn.close();
+
         return order;
     }
 
+    @Override
```
Comment (SerhiiPasko) [inline src/main/java/com/mateacademy/lessons15/OrderDaoImpl.java:87]: What is it?

File: src/test/java/com/mateacademy/lessons15/OrderDaoImplIntegrationTest.java [hunk 3]
```diff
@@ -13,10 +15,9 @@
 import org.junit.Test;
 import org.junit.runners.MethodSorters;
 
-
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```
Comment (SerhiiPasko) [inline src/test/java/com/mateacademy/lessons15/OrderDaoImplIntegrationTest.java:18]: Why do you use that?

File: src/test/java/com/mateacademy/lessons15/OrderDaoImplIntegrationTest.java [hunk 4]
```diff
@@ -34,4 +35,19 @@ public void testFindOrderById2() throws SQLException {
         System.out.println(order);
         assertNull(order);
     }
+
+    @Test
```
Comment (SerhiiPasko) [inline src/test/java/com/mateacademy/lessons15/OrderDaoImplIntegrationTest.java:39]: A;lso good to cheak result of both methods in one test

**Review by SerhiiPasko — APPROVED** [review]
Good. Read comments

**Pull request merged**